docs/usage.rst

```diff
@@ -738,7 +738,7 @@ So, the main difference is that output of ``.get()`` and ``.getall()`` methods
 is more predictable: ``.get()`` always returns a single result, ``.getall()``
 always returns a list of all extracted results. With ``.extract()`` method
 it was not always obvious if a result is a list or not; to get a single
-result either ``.extract()`` or ``.extract_first()`` should be called.
+result either ``.get()`` or ``.extract_first()`` should be called.
```

Comment:
The idea is the following: previously, to get a single result you had to call either .extract() - if it was a Selector, or .extract_first() - if it was a SelectorList. That's the point of introducing .get(), so that to get a single result you can always call .get().

Reply:
fair enough.. Do you know how to say it better?

Reply:
My suggstion would be to replace `should be called` by `needed to be called, depending if you had a Selector or SelectorList`.

Reply:
updated